src/modbus.c

```diff
@@ -137,8 +138,7 @@ static unsigned int compute_response_length_from_request(modbus_t *ctx, uint8_t
         /* Header + nb values (code from write_bits) */
         int nb = (req[offset + 3] << 8) | req[offset + 4];
         length = 2 + (nb / 8) + ((nb % 8) ? 1 : 0);
-    }
-        break;
+    } break;
```

Comment:
Since this is already "special", I find your version with the break on the same line a bit more difficult to read and understand. I'd prefer to leave it as is...